sksurv/tree/_criterion.pyx

```diff
@@ -315,13 +315,14 @@ cdef class LogrankCriterion(Criterion):
         dest[0] = ratio  # Nelson-Aalen estimator
         dest[1] = 1.0 - ratio  # Kaplan-Meier estimator
 
-        j = 2
+        j_delta = 3
```

Comment:
Changing from delta=2 to delta=3 only here seems to be insufficient i.e. probably some other changes somewhere are needed too but I haven't yet figured out what/where.